From 90f21a39277a8b4e00b41017f2f582c698571ceb Mon Sep 17 00:00:00 2001
From: Alison Wang <b18965@freescale.com>
Date: Fri, 28 Nov 2014 14:34:12 +0800
Subject: [PATCH] qspi: ls102xa: Adjust the size of QSPI AMBA Bus memory map

This patch adjust the size of QSPI AMBA Bus memory map to 64M
to avoid the issue that system default vmalloc size is not
enough.

Signed-off-by: Alison Wang <alison.wang@freescale.com>
Change-Id: Ie917be03b0a0479f2c49dc02b9c36cb04ecc87c8
Reviewed-on: http://git.am.freescale.net:8181/24671
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Zhengxiong Jin <Jason.Jin@freescale.com>
Reviewed-by: Richard Schmitt <richard.schmitt@freescale.com>
---
 arch/arm/boot/dts/ls1021a.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/ls1021a.dtsi b/arch/arm/boot/dts/ls1021a.dtsi
index 6a2634b..a091e22 100644
--- a/arch/arm/boot/dts/ls1021a.dtsi
+++ b/arch/arm/boot/dts/ls1021a.dtsi
@@ -136,7 +136,7 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 			reg = <0x0 0x1550000 0x0 0x10000>,
-				<0x0 0x40000000 0x0 0x10000000>;
+				<0x0 0x40000000 0x0 0x4000000>;
 			reg-names = "QuadSPI", "QuadSPI-memory";
 			interrupts = <GIC_SPI 131 IRQ_TYPE_LEVEL_HIGH>;
 			clock-names = "qspi_en", "qspi";
